### PR TITLE
adding docs for dmverity hashing and fixing markdown styling

### PR DESCRIPTION
**Pull request by SethHollandsworth**
also taking out "preview" annotation

### CODE REVIEW

File: src/confcom/azext_confcom/README.md [hunk 1]
```diff
@@ -37,66 +39,88 @@ Install the Azure CLI and Confidential Computing extension.
...
-    az extension list-available -o table | grep confcom
+```bash
+az extension list-available -o table | grep confcom
+```
 
 To add the most recent confcom extension, run:
 
-    az extension add --name confcom
+```bash
+az extension add --name confcom
```
Comment (hgarvison) [inline src/confcom/azext_confcom/README.md:49]: You could also add the update command here for when the users already have a version installed

File: src/confcom/azext_confcom/README.md [hunk 2]
```diff
@@ -143,123 +171,158 @@ Users just need to make a tar file by using the `docker save` command above, inc
...
-    allow_properties_access := false
-    allow_dump_stacks := false
-    allow_runtime_logging := false
-    allow_environment_variable_dropping := true
-    allow_unencrypted_scratch := false
+allow_properties_access := false
+allow_dump_stacks := false
+allow_runtime_logging := false
+allow_environment_variable_dropping := true
+allow_unencrypted_scratch := false
```
Comment (hgarvison) [inline src/confcom/azext_confcom/README.md:302]: Should add: https://msr-gryffindor.visualstudio.com/Taro/_wiki/wikis/GCS%20Policy%20Breakdown/47/framework?anchor=%60allow_capabilities_dropping%60#:~:text=allow_capabilities_dropping

File: src/confcom/azext_confcom/README.md [hunk 3]
```diff
@@ -143,123 +178,159 @@ Users just need to make a tar file by using the `docker save` command above, inc
...
 Because the ARM Template does not have a value defined for the "placeholderValue", the regular expression ".*" is used in the Rego policy. This allows for any value to be used. If the value is contained in a parameters file, that can be used when deploying such as:
 
-    az deployment group create --template-file "template.json" --parameters "parameters.json"
+```bash
+az deployment group create --template-file "template.json" --parameters "parameters.json"
+```
+
+## dmverity Layer Hashing
+
+To ensure the container that is being deployed is the intended container, the `confcom` tooling uses [dmverity hashing](https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/verity.html). This is done by downloading the container locally with the Docker Daemon (or using a pre-downloaded tar file of the OCI image) and performing the dmverity hashing using the [dmverity-vhd tool](https://github.com/microsoft/hcsshim/tree/main/cmd/dmverity-vhd). These layer hashes are placed into the Rego security policy in each container's "layers" field. Note that these dmverity layer hashes are different than the layer hashes reported by `docker image inspect`.
```
Comment (stevendongatmsft) [inline src/confcom/azext_confcom/README.md:267]: These layer hashes are placed into the Rego security policy in the "layers" field of each container.

File: src/confcom/azext_confcom/README.md [hunk 4]
```diff
@@ -143,123 +178,159 @@ Users just need to make a tar file by using the `docker save` command above, inc
...
+
+### Mixed-mode Policy Generation
+
+An OCI image can be made available for policy generation in three ways:
+
+1. The image is in the local Docker Daemon and can be found either with its image and tag names or sha256 hash.
```
Comment (stevendongatmsft) [inline src/confcom/azext_confcom/README.md:273]: The image is in the local Docker Daemon and can be found either with its image and tag names or with sha256 hash.

**Review by stevendongatmsft — COMMENTED** [review]
Looks pretty good. I only left 2 comments regarding the grammar.

**Review by stevendongatmsft — APPROVED** [review]
LGTM

**Pull request merged**